libvmi/driver/xen/xen_events_legacy.c

```diff
@@ -925,7 +947,7 @@ xen_set_mem_access_legacy(vmi_instance_t vmi, mem_access_event_t *event,
 
         hvmmem_access_t access = memaccess_conversion[page_access_flag];
 
-        rc = xen->libxcw.xc_hvm_set_mem_access(xch, dom, access, page_key, npages);
+        rc = xen->libxcw.xc_hvm_set_mem_access(xch, dom, access, gpfn, 1);
```

Comment:
Gotcha. It doesn't really worth a define but I'll add a comment to say that effectively ;)